org.mintguide.WebAppReader.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: org.mintguide.WebAppReader
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: /app/webappreader.sh
+finish-args:
+  - --device=dri
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+  - --share=network
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    QTWEBENGINE_DISABLE_SANDBOX: '1'
+modules:
+  - python3-requirements.json
+  - name: webappreader
+    subdir: app
+    buildsystem: simple
+    build-commands:
+      - cp webappreader.py /app/
+      - cp webappreader.sh /app/
+      - chmod +x /app/webappreader.sh
+      - cp constants.py /app/
+      - cp settings.py /app/
+      - cp tools.py /app/
+      - cp tools.py /app/
+      - cp -r ui /app/
+      - cp -r pages /app/
+      - cp -r database /app/
+      - cp -r src /app/
+    sources:
+      - type: dir
+        path: .
```

Comment:
this is not gonna work.
`dir` for the source is always incorrect.

Reply:
And just to clarify, here `.` **currently** only contains the manifest and surely should not contain the upstream source.